device.yml

```diff
@@ -325,18 +325,21 @@ registers:
     access: Write
     maskType: Channel3RangeConfig
     description: Selects the flow range for the channel 3.
-  Reserved0: &reserved
```

Comment:
I may be missing something, but all these changes are backwards compatible from the point of view of the API

Reply:
I think that all the changes are back compatible, so yes we can tag as 0.1.1.